src/components/shared/Terminal.tsx

```diff
@@ -12,11 +12,13 @@ function Terminal(prop: {
 }): JSX.Element {
   const [commands, setCommands] = useState<string[]>([]);
   const [input, setInput] = useState('$ ');
+  const [inputHistory, setInputHistory] = useState<string[]>([]);
+  const [inputHistoryIndex, setInputHistoryIndex] = useState(0);
 
   function executeList(flags: string, path: string): [string[], string[]] {
     const displayAll = flags.includes('a');
     const longFormat = flags.includes('l');
-    const fsObject = path.includes('/')
+    const fsObject = path.startsWith('/')
```

Comment:
Refactor - make these strings for path and file into constants for reusability

Reply:
Yup, better practice would be to add these into a constants file. Also, makes it easier to check which operations are supported at a glance. Will do this in a backlog PR.